### DISCUSSION TITLE
it shows all the time a bended bed - but its straight

### DISCUSSION THREAD

**Discussion by acrobat88** [jneilliii/OctoPrint-BedLevelVisualizer#390] [Q&A]
hi there,

first thanks for all the work! Nice plugin!

I have a strange problem: after i tested the bed its all the time shown as bended on the left side - even when i turn the Ultrabase 90 degree - it always bended on the left side and it doesnt change. It always shown like this:
![Bildschirmfoto 2021-02-23 um 18 24 50](https://user-images.githubusercontent.com/79534459/108882751-083c0f80-7605-11eb-8f68-6086e3039d2a.png)
Could it be a software bug?

**Comment by jneilliii** [comment]
Look at the actual data that is coming from your firmware weeken tou run the update mesh process. I'd suspect that it's showing the same thing.

**Reply by acrobat88** [reply]
What du you mean with "Look at the actual data that is coming from your firmware weeken tou run the update mesh process"?

**Reply by jneilliii** [reply]
Sorry, phone typo. Look at the actual points listed on the terminal tab when you finish probing. That is the data that the plugin uses to graph.
